setup.py

```diff
@@ -49,7 +49,7 @@
 
 extras_require = {
     'docs': [
-        'Sphinx>=1.4.2',
+        'Sphinx==1.4.4',
```

Comment:
Please keep it `>=1.4.2` and remove `:undoc-members:` that cause problems with autodoc module (anyway we should not have any undocumented functions/attributes).